region/maridia/inner-yellow.json

```diff
@@ -829,51 +829,70 @@
+                      "canTrickyDashJump",
+                      "canTrickyUseFrozenEnemies"
                     ]}
                   ]
                 },
                 {
-                  "name": "Maridia Bug Room Frozen Menu Bridge",
-                  "notable": true,
+                  "name": "Maridia Bug Room MidAir Morph with Gravity",
+                  "notable": false,
                   "requires": [
                     "Morph",
-                    "h_canNavigateUnderwater",
-                    "canTrickyUseFrozenEnemies"
+                    "Gravity",
+                    "canTrickyJump",
+                    {"or":[
+                      "HiJump",
+                      "h_canFly",
+                      "canWalljump",
+                      "canSpringBallJumpMidAir",
+                      "canTrickyDashJump",
+                      "canTrickyUseFrozenEnemies"
+                    ]}
                   ],
                   "note": [
-                    "Freeze one of the bugs (called Menus) in a position where you can use it to get into the morph passage.",
-                    "That allows passage without having to gain height while being morphed."
+                    "With Gravity, it's possible to move forward and mid-air morph into the morph passage.",
+                    "The momentum can push Samus into the tunnel, no need to press forward while in mid-air."
                   ]
                 },
                 {
                   "name": "Maridia Bug Room Suitless MidAir Morph",
                   "notable": false,
                   "requires": [
                     "h_canNavigateUnderwater",
-                    "h_canTwoTileJumpMorph"
+                    "h_canTwoTileJumpMorph",
+                    {"or":[
+                      "HiJump",
+                      "h_canFly",
+                      "canWalljump",
+                      "canSpringBallJumpMidAir",
+                      "canTrickyDashJump",
+                      "canTrickyUseFrozenEnemies"
+                    ]}
                   ]
                 },
                 {
-                  "name": "Maridia Bug Room MidAir Morph with Gravity",
+                  "name": "Maridia Bug Room Frozen Menu Bridge",
                   "notable": true,
                   "requires": [
                     "Morph",
-                    "Gravity"
+                    "h_canNavigateUnderwater",
+                    "canTrickyUseFrozenEnemies"
                   ],
                   "note": [
-                    "With Gravity, it's possible to move forward and mid-air morph into the morph passage.",
-                    "The momentum can push Samus into the tunnel, no need to press forward while in mid-air."
-                  ],
-                  "devNote": "FIXME This probably needs to be behind a tech as well?"
+                    "Freeze one of the bugs (called Menus) in a position where you can use it to get into the morph passage.",
```

Comment:
I can't tell how this is meant to be done.
Using two menus you can make a full bridge and jump on top and roll over, but its pretty impossible.
With one menu you can freeze it a little low and do a high jump morph to bounce from the bug to the tunnel.
The way I liked was freezing the menu just a little above the dry patch and waiting for it to unfreeze to damage boost into the hole.

In all cases, it was really hard to do this damageless (didn't try to)

Reply:
This strat was already in there, I just moved it. I never could get it, and thought it was going to be a lot harder than the midair morph option. I believe Map Rando ignores this notable - do you think we should remove the strat altogether?

Reply:
Ah, strat order got shuffled so I couldn't tell.

The strat should exist imo.  
I like the version where you stand in the water, take a hit for positioning, and freeze the bug near but left of samus.  Then morph on top and damage boost on the unfreeze.  And that would be 2 hits if all goes well.